web_src/js/svg.js

```diff
@@ -38,18 +40,33 @@ export const svgs = {
 
 const parser = new DOMParser();
 const serializer = new XMLSerializer();
+const parsedSvgs = new Map();
 
-// retrieve a HTML string for given SVG icon name, size and additional classes
-export function svg(name, size = 16, className = '') {
+function getParsedSvg(name) {
+  if (parsedSvgs.has(name)) return parsedSvgs.get(name).cloneNode();
+  const root = parser.parseFromString(svgs[name], 'text/html');
+  const svgNode = root.querySelector('svg');
+  parsedSvgs.set(name, svgNode);
+  return svgNode;
```

Comment:
I think we also need `cloneNode` here. Otherwise the first call of `svgNode('foo', 32)` will make the cached svg size becomes 32, it differs from the default value 16

Reply:
Hmm, I guess I will just remove the cache. Deep cloning a node is probably also pretty expensive, maybe even more then parsing.

Reply:
done